### PR TITLE
Changing ycsb status to working

### PR DESCRIPTION
**Pull request by aakarshg**

### CODE REVIEW

File: docs/ycsb.md [hunk 1]
```diff
@@ -16,7 +14,7 @@ Your resource file may look like this:
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: ycsb-couchbase-benchmark
+  name: ycsb-mongo-benchmark
```
Comment (jtaleric) [inline docs/ycsb.md]: Should you rename this to `docs/ycsb-mongo.md` to be specific that this is for mongodb?

File: tests/test_list [hunk 2]
```diff
@@ -4,3 +4,4 @@ test_iperf3.sh
 test_sysbench.sh
 test_uperf.sh
 test_byowl.sh
+test_ycsb.sh
```
Comment (jtaleric) [inline tests/test_list]: Should the testname reflect the database?

File: docs/ycsb.md (hunk 1 above)
Comment (aakarshg) [reply docs/ycsb.md]: Only the example here is specifically for running against mongodb, so I dont think so.

File: tests/test_list (hunk 2 above)
Comment (aakarshg) [reply tests/test_list]: We're not testing the database at all here, all the test tries to see if ycsb can actually run. For sanity purposes it'll load the database.

File: docs/ycsb.md (hunk 1 above)
Comment (jtaleric) [reply docs/ycsb.md]: The remove mongo?

**Pull request merged**